src/models/ConstrainedMetaModel.ts

```diff
@@ -131,20 +132,6 @@ export class ConstrainedEnumModel extends ConstrainedMetaModel {
     public values: ConstrainedEnumValueModel[]) {
     super(name, originalInput, type);
   }
-
-  getNearestDependencies(): ConstrainedMetaModel[] {
-    let dependencyModels = Object.values(this.values).filter(
-      (enumModel) => {
-        return enumModel.value instanceof ConstrainedReferenceModel;
-      }
-    ).map((enumModel) => {
-      return enumModel.value as ConstrainedReferenceModel;
-    });
-
-    //Ensure no duplicate references
-    dependencyModels = [...new Set(dependencyModels)];
-    return dependencyModels;
-  }
```

Comment:
This is removed because enums are not supposed to have ConstrainedMetaModels i.e. they will never have dependencies, they are pure values.